### DISCUSSION TITLE
Can we have multiselect row options for grafana table panel

### DISCUSSION THREAD

**Discussion by a-imran99** [grafana/grafana#46160] [Dashboards, Panels & Visualizations]
How can we achieve multiselect row option in grafana new table panel. Is it possible to modify the react-table dependency and add useRowSelect to add checkboxes to select row?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
